user-guide/Advanced_Modules/DOM/tutorials/DOM_Remove_FieldDescriptor_Definition.md

```diff
@@ -0,0 +1,237 @@
+                        if (softDeletedFieldValues.Any())
+                        {
+                            softDeletedFieldValues.ForEach(one => section.RemoveFieldValueById(one.FieldDescriptorID));
+                            instancesToUpdate.Add(instance);
+                        }
+                    }
+                }
+            }
+
+            engine.GenerateInformation($"Updating {instancesToUpdate.Count} DOM instances");
+            instancesToUpdate.ForEach(one => domHelper.DomInstances.Update(one));
+        }
+    }
+}
+```
+
+After you have implemented this, the phone number for each person should no longer be visible in your low-code app.
+
+![Step3](~/user-guide/images/DOM_Remove_FieldDescriptor_Definition_Step3.png)
+
+## Step 4: Delete the field descriptor from the section definition
+
+Once the field descriptor has been soft-deleted and you have made sure that it is not used in any DOM instances, you can delete the field descriptor from the section definition.
+
+Use the code provided below to accomplish the following:
+
+- The `FieldDescriptor` is deleted from the `SectionDefinition`.
+
+- The `SectionDefinition` is updated.
+
+```C#
+using System.Linq;
+using Skyline.DataMiner.Automation;
+using Skyline.DataMiner.Net.Apps.DataMinerObjectModel;
+using Skyline.DataMiner.Net.Messages.SLDataGateway;
+using Skyline.DataMiner.Net.Sections;
+
+namespace Tutorial
+{
+    ///<summary>
+    /// This script will remove the FieldDescriptor and update the SectionDefinition. 
+    ///</summary>
+    public class Script
+    {
+        private readonly string _moduleName = "people_app";
+        private readonly string _sectionDefinitionName = "PeopleSectionDefinition";
+
+        public void Run(Engine engine)
+        {
+            var domHelper = new DomHelper(engine.SendSLNetMessages, _moduleName);
+
+            var peopleSectionDefinition = domHelper.SectionDefinitions.Read(
+                SectionDefinitionExposers.Name.Equal(_sectionDefinitionName)).FirstOrDefault() as CustomSectionDefinition;
+            if (peopleSectionDefinition == null)
+            {
+                engine.GenerateInformation($"Did not find the '{_sectionDefinitionName}'");
+                return;
+            }
+
+            var softDeletedFieldDescriptor = peopleSectionDefinition.GetAllFieldDescriptors().FirstOrDefault(one => one.IsSoftDeleted);
+
+            if (softDeletedFieldDescriptor == null)
+            {
+                engine.GenerateInformation("Did not find the soft-deleted FieldDescriptor");
+                return;
+            }
+
+            peopleSectionDefinition.RemoveFieldDescriptor(softDeletedFieldDescriptor.ID);
+            domHelper.SectionDefinitions.Update(peopleSectionDefinition);
+        }
+    }
+}
+```
+
+Optionally, you can inspect the `SectionDefinition` with the [SLNetClientTest tool](xref:Opening_the_SLNetClientTest_tool) to check if the phone number is now completely removed:
```

Comment:
This last action needs more detail, as most users will not know how to do this (and the SLNetClientTest tool is not something they should just experiment with, give the massive impact this could have on their system). Perhaps you can change this into an optional extra step, and then detail the different actions a user has to take in order to do this?